src/controller/replication/execution.go

```diff
@@ -109,10 +109,33 @@ func (c *controller) Start(ctx context.Context, policy *replicationmodel.Policy,
 	if op := operator.FromContext(ctx); op != "" {
 		extra["operator"] = op
 	}
+
 	id, err := c.execMgr.Create(ctx, job.ReplicationVendorType, policy.ID, trigger, extra)
 	if err != nil {
 		return 0, err
 	}
+
+	// If running executions are found, skip the current execution and mark it as skipped.
+	if policy.SingleActiveReplication {
```

Comment:
Why do we create an empty execution (without any tasks) for a skipped action? Is it only for the UI to show a "skipped" status?

Shouldn't we return the currently running execution ID with an HTTP 409 response to the API caller, instead of creating a new execution record with no actual processing?